pkg/kn/commands/service/export.go

```diff
@@ -230,3 +302,13 @@ func revisionListSortFunc(revisionList *servingv1.RevisionList) func(i int, j in
 		return a.Name > b.Name
 	}
 }
+
+func stripGeneratedFieldsfromService(svc *servingv1.Service) {
+	delete(svc.ObjectMeta.Annotations, "serving.knative.dev/creator")
```

Comment:
Maybe we can the annotations that should be removed into an array and put it top-level in this file with a prominent comment what this array is about? This probably makes maintaining this list easier (as I expect it will require some maintenance and/or fine-tuning)